### DISCUSSION TITLE
Is there any plan for a Rust SDK?

### DISCUSSION THREAD

**Discussion by hut36** [powersync-ja/powersync-js#433] [Q&A]
I am developing an app based on Tauri, it would be nice if there is a Rust SDK.

**Comment by rkistner** [comment]
There are no specific plans for a Rust SDK just yet.

We do have plans to make our [SQLite extension](https://github.com/powersync-ja/powersync-sqlite-core/) APIs stable and documented, which would make it easier to integrate into environments such Tauri yourself. However, we don't have any timelines for this yet.

**Comment by basicserge** [comment]
This is something I'm also missing a lot as well. Having [this issue](https://github.com/tauri-apps/tauri/discussions/12536) with the web sdk using Tauri at the moment.

**Comment by cahofmeyr** [comment]
Please add your vote to the roadmap here: https://roadmap.powersync.com/c/133-c-c-sdk :)

**Comment by cahofmeyr** [comment]
ICYMI:

https://releases.powersync.com/announcements/introducing-the-powersync-rust-client-sdk-experimental

https://docs.powersync.com/client-sdks/reference/rust
